### ISSUE TITLE
Map icons: Improved consistency

### ISSUE THREAD

**Issue by metoxys** [PLTytus/GTAWeb#70]
**Describe your idea:**  
Right now, multiple options behave inconsistently when routes are involved. For instance, Stash Houses and G's Caches have multiple potential spawn locations, of which only one is active at a time. Accordingly, they are ignored when it comes to route calculation.  
If that is the case, why does the same logic not apply to things like Exotic Export spawn locations and Madrazo Hit locations?

I suggest making all things with variable spawn locations ignored when it comes to route calculations.

**Comment by metoxys** [comment]
> Stash Houses and G's Caches location is visible on your in-game map at yours specific location. So path is not needed because you do not need to look for it. The only thing you may need from them on map is screenshoots showing the exact location, all possible safe codes for Stash House if you can't find it, or information about collectible.

Then why are Exotic Export spawn locations and Madrazo Hit locations pathed? It makes very little logical sense  
![path](https://github.com/user-attachments/assets/2360d8f0-2fa7-4741-86ae-1de0215ade0a)

**Comment by PLTytus** [comment]
- Path for Exotic Exports is to find best way to look for currently spawned vehicle.
- Path for Madrazo Hits is to find your location. Your location for Madraza Hit is only known when you create a new session on your own. Best is to create new invite, friends or crew only session - and then go to yellow marker for Madrazo Hit.

**Comment by PLTytus** [comment]
I can offer a solution I offered in https://github.com/PLTytus/GTAWeb/issues/64#issuecomment-2159364127:

> I might try adding an "exclude/include group from path" feature if you're happy with this kind of solution - let me know if I should add it to the todo-list.

**Comment by metoxys** [comment]
Please do, it seems to be the best option

**Comment by PLTytus** [comment]
Ok, I will continue this topic in https://github.com/PLTytus/GTAWeb/issues/71.

**Comment by PLTytus** [comment]
The highlighted line on screenshot is wrong - I had wrong intel about this content and I deleted this misinformation. The location should be visible no matter where you are on map (same as Stash House and G's Cache) - so I disabled "plote a route" for this. But I am still gonna proceed with "exclude/include group from path" feature.
![image](https://github.com/user-attachments/assets/ee560c36-c4bf-43bf-922c-a007f800da9a)
